README.md

```diff
@@ -1,42 +1,104 @@
-Useful commands for [CopyQ clipboard manager](https://github.com/hluk/CopyQ).
+# CopyQ Command Project
 
-You can share your commands and ideas here.
-Just open pull request or an issue.
+## 🎯 Goal
 
-# Categories
```

Comment:
I would prefer to keep this section instead of the new "Directory Structure". I don't think it is necessary to document every directory/file.